projects/fhi-angular-components/src/lib/fhi-tree-view-selection/fhi-tree-view-selection-item.model.ts

```diff
@@ -1,8 +1,8 @@
-export interface FhiTreeViewSelectionItem {
+import { FhiTreeViewSelectionItemInternal } from './fhi-tree-view-selection-item-internal.model';
+
+export interface FhiTreeViewSelectionItem
+  extends Omit<FhiTreeViewSelectionItemInternal, 'internal'> {
   children?: FhiTreeViewSelectionItem[];
   id?: number | string;
-  isChecked?: boolean;
-  isExpanded?: boolean;
-  hasCheckedDescendant?: boolean;
-  name: string;
+  [key: string]: unknown;
```

Comment:
Jeg tenkte først at det ville være "nice" å tillate alle mulige properties på dette objektet, men siden vi nå uansett må beholde `id` for å unngå breaking, og siden dette introduserer ett lite problem hva gjelder property `internal` ([se logs her](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo/pull/778/files#diff-11d670e3ec9c441ff43d8a1900a9ad4a15e8cbebd989eb8325ecea1b72f8af0cR33)); er det egentlig best å droppe `[key: string]: unknown;`?  
Hva tenker dere @VillereV, @bittakv ?